osu.Game/Tests/Platform/TestStorage.cs

```diff
@@ -7,13 +7,11 @@ namespace osu.Game.Tests.Platform
 {
     public class TestStorage : DesktopStorage
     {
-        public TestStorage(string baseName) : base(baseName, null)
+        public TestStorage(string baseName)
+            : base(baseName, null)
         {
         }
 
-        public override string GetDatabaseConnectionString(string name)
-        {
-            return "DataSource=:memory:";
-        }
+        public override string GetDatabaseConnectionString(string name) => "Data Source=" + GetUsablePathFor($"{(object)name}.db", true);
```

Comment:
`(object)name` ?

Reply:
weird, that's how it is in the other class. probably left over from some old usage pattern